org.btchd.btchd-qt.json

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "org.btchd.btchd-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "btchd-qt",
+  "finish-args": [
+    /* Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network",
+    /* Tor auth cookie */
+    "--filesystem=/run/tor/control.authcookie:ro",
+    "--filesystem=/var/run/tor/control.authcookie:ro"
+  ],
+  "modules": [
+    {
+      "name": "bitcoinhd-core",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/btchdd",
+        "install -m 0744 -D -t /app/bin/internal bin/btchd-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/btchd-cli"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/btchd/btchd/releases/download/v1.4.3/bhd-v1.4.3-7fe720529-x86_64-linux-gnu.tar.gz",
+          "sha256": "d5c1887693c2f9d56f44476315ac2f7c68c5ac9751d23c1b988b3ea6e15108bf"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["i386"],
+          "url": "https://github.com/btchd/btchd/releases/download/v1.4.3/bhd-v1.4.3-7fe720529-i686-linux-gnu.tar.gz",
+          "sha256": "f9a435277a30ee075da989bf13fdfaa4b2c6fd20e274252c6936acf9721f281f"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["arm"],
+          "url": "https://github.com/btchd/btchd/releases/download/v1.4.3/bhd-v1.4.3-7fe720529-arm-linux-gnueabihf.tar.gz",
+          "sha256": "1e5abee5de2533fb60ada173314fdd710692ac6ec41d0076b71585202295b331"
+        },
```

Comment:
neither i386 nor arm are supported by the runtime, so this is pointless.

Reply:
Removed.